### DISCUSSION TITLE
Save in Streamlit

### DISCUSSION THREAD

**Discussion by MichaelSeitz98** [Kanaries/pygwalker#221] [Show and tell]
Great tool! I am quite new to open source development but I already love it. 

I was wondering if it is possible to update the config/spec in the Streamlit env in the same way as it is possible via the "normal" jupyter env? 
Any suggestions / ideas / workaround? Or is there already something in the pipeline?

Thanks a lot!
Michael

**Comment by longxiaofei** [comment]
Hi, Due to Streamlit don't support request-response communication, so we may use hack way to finish it,  this feature is currently planned, but there is no clear publish time currently.

**Reply by MichaelSeitz98** [reply]
Alright! Thanks a lot for your answer! I will keep an eye on the releases ;)
